lib/public.js

```diff
@@ -150,7 +150,7 @@ class Public {
             id: id,
             properties: properties,
             customerProperties: customerProperties,
-            timestamp: timestamp,
```

Comment:
This should actually remain as `timestamp` - this data is being passed to the `track` function. Full context for this change:
https://github.com/klaviyo/node-klaviyo/blob/f1707b88a7eb3581511649021570e685ede3d892/lib/public.js#L147-L156

Unless we want to change the kwarg name to time, this should stay `timestamp`